### ISSUE TITLE
Sometimes placeholder values change - this doesn't seem to update the checksum value

### ISSUE THREAD

**Issue by sjd2021** [lecaillon/Evolve#269]
Logically, I guess the placeholders come into play after a checksum has already been calculated. How can I ensure the checksum changes so that my repeatable migrations still run when needed without me having to make a random whitespace change?

**Comment by lecaillon** [comment]
Yes, you're right. That's why it is not re-executed even if you change the placeholders. Maybe you can use the latest version of Evolve `3.1.0-alpha7` and use the new option `-- evolve-repeat-always` at the beginning of the script to always execute a repeatable migration ?

**Comment by sjd2021** [comment]
I don't want that to happen since we run so many instances. Is it worth exploring an option to calculate the checksums post-placeholder-eval or is such an idea not likely to be approved due to philosophical reasons, etc.?

**Comment by matthooper** [comment]
+1 for checksums after placeholder substitution. Or at least having a way to denote some scripts as repeat-if-changed vs repeat-always.
